proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -35,6 +35,35 @@ message BreakingChangePolicy {
   BreakingChangeCategory category = 3;
 }
 
+message UniquenessPolicy {
+  optional bool enabled = 1;
+}
+
+message UniquenessState {
+  bool has_collisions = 1;
+  repeated Collision collisions = 2;
+}
+
+message Collision {
+  string id = 1;
+  CollisionType type = 2;
+  string reference = 3;
+  repeated CollisionSource sources = 4;
+}
+
+message CollisionSource {
+  string id = 1;
```

Comment:
do we need the `id`s here and for `Collision`? my only hesitation is that if I resolve a collision and then later introduce "the same" collision, we aren't going to be assigning stable IDs, so I'm curious if you see a use-case for them. (happy to leave them in, just wondering!)

Reply:
+1